src/psyclone/tests/psyir/backend/sir_test.py

```diff
@@ -225,9 +225,12 @@ def test_sirwriter_node_1():
     '''
     schedule = get_routine(CODE)
 
-    class Unsupported(Node):
+    class Unsupported(ScopingNode):
         '''A PSyIR node that will not be supported by the SIR writer but
         accepts any children inside.'''
+        def __init__(self):
+            super().__init__()
+
```

Comment:
The empty constructor shouldn't be needed.

And why did you need to subclass ScopingNode here?

Reply:
We're adding a Routine to an `Unsupported`, so the Routine's parent needs to have a symbol table (i.e. be a ScopingNode or similar).